maven-plugin/src/main/java/org/openmrs/maven/plugins/AddFeature.java

```diff
@@ -0,0 +1,90 @@
+package org.openmrs.maven.plugins;
+
+import edu.emory.mathcs.backport.java.util.Arrays;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugin.logging.SystemStreamLog;
+import org.openmrs.maven.plugins.utility.OwaHelper;
+import org.openmrs.maven.plugins.utility.Project;
+import org.openmrs.maven.plugins.utility.SDKConstants;
+import org.openmrs.maven.plugins.utility.XmlHelper;
+
+import java.io.File;
+import java.util.List;
+import java.util.Properties;
+
+import static org.twdata.maven.mojoexecutor.MojoExecutor.artifactId;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.executeMojo;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.executionEnvironment;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.goal;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.groupId;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.plugin;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.version;
+
+/**
+ * @goal add-feature
+ * @requiresProject false
+ */
+public class AddFeature extends AbstractTask {
+
+    public static final String OPEN_WEB_APP = "Open Web App";
+    public static final String[] OPTIONS = {OPEN_WEB_APP};
+
+    /**
+     * feature user wants to add
+     *
+     * @parameter expression="${feature}"
+     */
+    private String feature;
+
+    @Override
+    public void executeTask() throws MojoExecutionException, MojoFailureException {
+        if (Project.hasProject(new File(System.getProperty("user.dir")))) {
+            feature = wizard.promptForMissingValueWithOptions("What feature would you like to add?", feature, "feature", Arrays.asList(OPTIONS), null, null);
+            if(feature.equals(OPEN_WEB_APP) || feature.equals("owa")){
+                addOwaSubmodule();
+            } else {
+                throw new IllegalArgumentException("Adding feature "+feature+" is not available. Available features: "+OPTIONS);
+            }
+        } else {
+            throw new IllegalArgumentException("No project found in this directory. Please enter project's main directory and run this command again");
+        }
+    }
+
+    private void addOwaSubmodule() throws MojoExecutionException {
+        //apply changes to config.xml and main pom.xml
+        wizard.showMessage("Modifying pom.xml files...");
+        new XmlHelper().modifyXml(new File(mavenProject.getBasedir(), "omod"+File.separator+"pom.xml"), "archetype-submodule-owa/omod.pom.xml");
+        new XmlHelper().modifyXml(new File(mavenProject.getBasedir(), "omod"+File.separator+"src"+File.separator+"main"+File.separator+"resources"+File.separator+"config.xml"), "archetype-submodule-owa/config.xml");
+
```

Comment:
Shouldn't the second argument of XmlHelper.modifyXml() be passed with File.separator instead of hardcoded "/"